src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -459,9 +460,9 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
                             className="dc__link"
                             href={DOCUMENTATION.GLOBAL_CONFIG_GITOPS}
                         >
-                            {' '}
-                            Learn more about GitOps{' '}
-                        </a>{' '}
+                            &nbsp;
```

Comment:
Expecting You wanted these spaces and were not because of the formatting issue right?